src/main/src/test/java/org/geoserver/wfsng/WfsCompatibilityTest.java

```diff
@@ -67,12 +67,19 @@ public void testWfsCompatitibility() throws IOException {
         }
                 
         assertTrue(store instanceof WFSDataStore);
-                
-        FeatureType type = ftInfo.getFeatureType();
         
-        assertEquals("sf_archsites", type.getName().getLocalPart());        
+        try{
```

Comment:
so, like this?
`try{
            FeatureType type = ftInfo.getFeatureType();
            assertEquals("sf_archsites", type.getName().getLocalPart());
            assertEquals("sf_archsites", ftInfo.getFeatureSource(null, null).getName().getLocalPart());
        } catch(IOException e){
            String expectedMessage = "Schema 'sf_archsites' does not exist.";
            assertEquals("Exception message must be correct", expectedMessage, e.getMessage());
        }`